.dockerignore

```diff
@@ -0,0 +1,11 @@
+# Remove all, we'll include only the necessary folders via !
+*
+
+!sunshine
+!cmake
+!assets
+!scripts
+!third-party
+!CMakeLists.txt
+!*.in
+!LICENSE
```

Comment:
Please add a blank line.

Also, do we need the `sunshine.png` or `sunshine.ico` ? Might be easier to exclude the folders and files we don't want and might be less likely to break?